internal/api/grpc/management/login_policy.go

```diff
@@ -26,15 +26,15 @@ func (s *Server) GetDefaultLoginPolicy(ctx context.Context, _ *empty.Empty) (*ma
 }
 
 func (s *Server) CreateLoginPolicy(ctx context.Context, policy *management.LoginPolicyRequest) (*management.LoginPolicy, error) {
-	result, err := s.command.AddLoginPolicy(ctx, loginPolicyRequestToDomain(ctx, policy))
+	result, err := s.command.AddLoginPolicy(ctx, authz.GetCtxData(ctx).OrgID, loginPolicyRequestToDomain(ctx, policy))
```

Comment:
we should discuss how we want to pass the orgID / RO (separate param or in domain object)

Reply:
we have it every where as separate param so i thought we should make this here as well